cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/PathBuilder.java

```diff
@@ -61,12 +61,29 @@ public PathBuilder addParameter(CruiseControlParameters param, List<String> valu
         return this;
     }
 
+    private void addIfNotZero(PathBuilder builder, CruiseControlParameters param, int value) {
+        if (value > 0) {
+            builder.addParameter(param, String.valueOf(value));
+        }
+    }
+
+    private void addIfNotZero(PathBuilder builder, CruiseControlParameters param, long value) {
+        if (value > 0) {
+            builder.addParameter(param, String.valueOf(value));
+        }
+    }
```

Comment:
can you have just the version with `long` and using cast to `int` of the parameter when it's safe to do it?